### PR TITLE
Add printing to remaining classes

### PR DESCRIPTION
**Pull request by lazappi**
Add `print()` and `to_string()` methods to the remaining R6 classes. Output tries to replicate the Python formatting except for some cases where the corresponding Python object doesn't have a print output.

### CODE REVIEW

**Comment by lazappi** [comment]
## Examples

<details><summary>Artifact</summary>
<p>

#### Python

image

#### R

image

</p>
</details> 

<details><summary>Field</summary>
<p>

#### Python

image

#### R

image

</p>
</details>

<details><summary>Instance</summary>
<p>

#### R

image

</p>
</details> 

<details><summary>InstanceAPI</summary>
<p>

#### R

image

</p>
</details>

<details><summary>InstanceSettings</summary>
<p>

#### R

image

</p>
</details>

<details><summary>Module</summary>
<p>

#### R

image

#### R (`to_string()`)

image

</p>
</details>

<details><summary>Record</summary>
<p>

#### Python

image

#### R

image

</p>
</details>

<details><summary>Registry</summary>
<p>

#### Python

image

#### R

image

#### R (`to_string()`)

image

</p>
</details>

File: README.md [hunk 1]
```diff
@@ -50,30 +50,7 @@ db <- connect("laminlabs/cellxgene")
...
-        Run: active binding
-        Storage: active binding
-        Transform: active binding
-        ULabel: active binding
-        User: active binding
-      Private:
-        .api: API, R6
-        .module_classes: list
-        .settings: InstanceSettings, R6
+    cellxgene Instance(modules='c('core', 'bionty')')
```
Comment (rcannood) [inline README.md:53]: This should probably say something like `cellxgene Instance(modules=c('core', 'bionty'))`.

Given that there is currently no documentation on what you can or cannot do with an Instance class, could we shed some light on:

* The instance core classes
* An instance's extra modules (e.g. bionty, wetlab)
* A list of functions uses can call

Comment (rcannood) [reply README.md:53]: Or maybe this is something that should be part of something like `instance$describe()`?

Comment (lazappi) [reply README.md:53]: I've changed it to the longer format I was using for `Module` and `Registry`. I think that should show enough info now.

File: R/Instance.R [hunk 2]
```diff
@@ -127,17 +170,47 @@ Instance <- R6::R6Class( # nolint object_name_linter
...
+          "]"
         )
       )
 
-      field_strings <- make_key_value_strings(fields)
+      module_names <- self$get_module_names()
+      module_names <- module_names[module_names != "core"]
+
+      if (length(module_names) > 0) {
+        mapping["AdditionModules"] = paste0(
```
Comment (rcannood) [inline R/Instance.R]:
```suggestion
        mapping["AdditionModules"] <- paste0(

Comment (lazappi) [reply R/Instance.R]: I fixed this already 😸

Comment (rcannood) [reply R/Instance.R]: :+1:

**Review by rcannood — APPROVED** [review]
LGTM!

**Pull request merged**